tests/test_2024_11_26.py

```diff
@@ -0,0 +1,379 @@
+        #
+        # 3: else if (4) then (5) else (6)
+        Param(
+            SpecialArgumentID.LOGIC_OP_PARAM_ID, Op.IF_ELSE, encode_argument_value_if(condition=4, success=5, failure=6)
+        ),
+        # 4: (_token == DAI)
+        Param(token_arg_index, Op.EQ, ArgumentValue(dai_limit.address)),
+        # 5: { return _amount <= 2_000_000 }
+        Param(amount_arg_index, Op.LTE, ArgumentValue(dai_limit.limit)),
+        #
+        # 6: else if (7) then (8) else (9)
+        Param(
+            SpecialArgumentID.LOGIC_OP_PARAM_ID, Op.IF_ELSE, encode_argument_value_if(condition=7, success=8, failure=9)
+        ),
+        # 7: (_token == LDO)
+        Param(token_arg_index, Op.EQ, ArgumentValue(ldo_limit.address)),
+        # 8: { return _amount <= 5_000_000 }
+        Param(amount_arg_index, Op.LTE, ArgumentValue(ldo_limit.limit)),
+        #
+        # 9: else if (10) then (11) else (12)
+        Param(
+            SpecialArgumentID.LOGIC_OP_PARAM_ID,
+            Op.IF_ELSE,
+            encode_argument_value_if(condition=10, success=11, failure=12),
+        ),
+        # 10: (_token == USDC)
+        Param(token_arg_index, Op.EQ, ArgumentValue(usdc_limit.address)),
+        # 11: { return _amount <= 2_000_000 }
+        Param(amount_arg_index, Op.LTE, ArgumentValue(usdc_limit.limit)),
+        #
+        # 12: else if (13) then (14) else (15)
+        Param(
+            SpecialArgumentID.LOGIC_OP_PARAM_ID,
+            Op.IF_ELSE,
+            encode_argument_value_if(condition=13, success=14, failure=15),
+        ),
+        # 13: (_token == USDT)
+        Param(token_arg_index, Op.EQ, ArgumentValue(usdt_limit.address)),
+        # 14: { return _amount <= 2_000_000 }
+        Param(amount_arg_index, Op.LTE, ArgumentValue(usdt_limit.limit)),
+        #
+        # 15: else if (16) then (17) else (18)
+        Param(
+            SpecialArgumentID.LOGIC_OP_PARAM_ID,
+            Op.IF_ELSE,
+            encode_argument_value_if(condition=16, success=17, failure=18),
+        ),
+        # 16: (_token == ETH)
+        Param(token_arg_index, Op.EQ, ArgumentValue(eth_limit.address)),
+        # 17: { return _amount <= 1000 }
+        Param(amount_arg_index, Op.LTE, ArgumentValue(eth_limit.limit)),
+        #
+        # 18: else { return false }
+        Param(SpecialArgumentID.PARAM_VALUE_PARAM_ID, Op.RET, ArgumentValue(0)),
+    ]
+
+def test_vote(helpers, accounts, vote_ids_from_env, stranger):
+
+    # misc
+    easy_track = interface.EasyTrack("0xF0211b7660680B49De1A7E9f25C65660F0a13Fea")
+    nor = contracts.node_operators_registry
+    prepare_agent_for_usdc_payment(15_000_000 * (10**6))
+    prepare_agent_for_steth_payment(20_000 * 10**18)
+    EVM_SCRIPT_EXECUTOR = "0xFE5986E06210aC1eCC1aDCafc0cc7f8D63B3F977"
+    perm_manager = contracts.acl.getPermissionManager(contracts.finance, convert.to_uint(Web3.keccak(text="CREATE_PAYMENTS_ROLE")))
+    contracts.acl.grantPermissionP(
```

Comment:
this is a strange thing. Could you explain why do we need to grant the executor the role in tests?
It seems that it's either not needed or essentially wrong

Reply:
@zuzueeka i do not see that we geant the role here - we set elevated permissions (for the amount of stETH specifically) and that's all
Here I allow to transfer 5k stETH (instead of 2k stETH) at once in order to make fewer motions to transfer stETH. Too much motions in this test will result in the chain moving to the next period (after Jan 1st 2025)

Reply:
> Too much motions in this test will result in the chain moving to the next period (after Jan 1st 2025)

theoretically (need to check) you can start motions in parallel